### DISCUSSION TITLE
invalid requested action when using `getAuth(window._firebaseApp)`

### DISCUSSION THREAD

**Discussion by teaglebuilt** [firebase/firebase-js-sdk#7297] [Q&A]
Is there any use case on using firebase in a micro frontend environment? 

The approach i took was a singleton below

```ts
let firebaseInstance: FirebaseApp;

export const initializeFirebase = (): FirebaseApp => {
  if (window._firebaseApp === undefined) {
    console.log(`BCST : window._firebaseApp = undefined,
        setting window._firebaseApp `);
    console.log(appConfig.firebase);
    const firebaseInstance = initializeApp(appConfig.firebase);
    window._firebaseApp = firebaseInstance;
  }
  return firebaseInstance;
};
```

However, for some reason, when i use `getAuth(window._firebaseApp)`, the app is there on the window but i get an error during the sign in process that the requested action is invalid.

**Comment by hsubox76** [comment]
I don't see a "requested action is invalid" error string in our codebase. I do see that it's an error for signing in using Google login with a popup. I'm not sure exactly what "micro frontend environment" you're using but does it block popups? Does it use a special url?

The first thing I found about that error was a Stack Overflow where the issue was they didn't authorize the domain https://stackoverflow.com/questions/69207851/the-google-sign-in-popup-doesnt-last-long-or-it-says-the-requested-action-is-i 

So maybe your micro frontend has a domain that hasn't been authorized in the console?
